specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/examples/expressRouteProviderPortList.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "portPairDescriptor": "bvtazureixpportpair1",
```

Comment:
this is not an appropriate top level property.  It should be in the "properties" object

Reply:
will remove this.